_includes/post-metadata.html

```diff
@@ -3,7 +3,7 @@
 <div class="h6">
   <span class="mr1 nowrap">
     <i class="fa fa-calendar"></i>
-    <time datetime="{{ include.post.date | date:"%Y-%m-%d" }}">{{ include.post.date | date:"%b %d, %Y" }}</time>
+    <time datetime="{{ include.post.date | date:"%Y-%m-%d" }}">{{ include.post.date | date:"%b %-d, %Y" }}</time>
```

Comment:
It's in the git commit. Changes "Oct 05, 2016" to "Oct 5, 2015" (no more leading zeros)